doc/source/migrations/index.rst

```diff
@@ -25,6 +25,13 @@ Development version
   ``ansys/actions/doc-deploy-changelog``, ``ansys/actions/doc-deploy-dev``, and
   ``ansys/actions/doc-deploy-stable``.
 
+**Deprecated features:**
+
+- The ``ansys/actions/doc-deploy-index`` action has been deprecated and will be removed in the next release.
+  With the deprecation of ``pymeilisearch`` and the adoption of a static search index via the ``ansys-sphinx-theme``,
+  the ``ansys/actions/doc-deploy-index`` action is no longer necessary.
+
+
```

Comment:
The ``ansys/actions/commit-style`` action has been renamed to ``ansys/actions/check-pr-title``.